tmp/psmdb/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG PSMDB_VERSION
+FROM percona/percona-server-mongodb:$PSMDB_VERSION
+
+USER root
+ADD https://github.com/percona/mongodb-orchestration-tools/releases/download/0.4.1/k8s-mongodb-initiator /usr/local/bin
+ADD https://github.com/percona/mongodb-orchestration-tools/releases/download/0.4.1/mongodb-healthcheck /usr/local/bin
+ADD entrypoint.sh /
```

Comment:
I hope we replaced original entrypoint due on purpose

Reply:
Yes, it's the same entrypoint.sh with 1 additional line to set the mode on the auth keyfile. I wanted to do this in the "Command" field in the Golang code but I could not get it to work - adding 'install' commands before the entrypoint breaks the arg list that is passed to mongod.